03/app.js

```diff
@@ -3,34 +3,60 @@ import ReactDOM from 'react-dom';
 
 class Article extends React.Component {
     state = {
+        comment: '',
         comments: [],
     }
     
     render() {
         const {title, body} = this.props;
+        const {comment} = this.state;
         return (
             <article>
                 <h1>{ title }</h1>
                 <p>{ body }</p>
-                <section>
-                    <form>
+                <section >
+                    <form onSubmit ={this.submitHandler}>
                         <div>
                             <label>
                                 <textarea 
-                                    style={{ "min-width": "300px", "min-height": "120px" }} 
-                                    name="content" 
+                                    style={{ "minWidth": "300px", "minHeight": "120px" }} 
+                                    name="content" value ={comment} onChange = {this.changeComment} 
                                 />
                             </label>
                         </div>
                         <div><input type="submit" value="dodaj komentarz" /></div>
                     </form>
                     <ul>
-                        {/* tutaj komentarze jako <li/>, ps. tak wygląda komentarz do kodu w JSX */}
+                        {this.renderCommentsList()}
                     </ul>
                 </section>
             </article>
         )
     }
+    renderCommentsList () {
+        const {comments} = this.state;
+        return comments.map(item => {
+            return (
+                <li>{item}</li> )
+        })
+    }
+
+    changeComment = e => {
+        this.setState({comment: e.target.value})
+    }
+
+    submitHandler = e => {
+        e.preventDefault();
+        const {comment} = this.state;
+        if (comment.length > 5) {
+            this.addComment(`${comment}`);
+            this.setState({comment: ''})
+        } else {alert('Comment should contain at least 5 characters')}
+    }
+
+    addComment = (item) => {
+        this.setState({comments: [...this.state.comments, item]})
```

Comment:
Można też tutaj zresetować od razu komentarz tj. `this.setState({comments: [...this.state.comments, item], 'comment': ''})`

Reply:
Zmienione :)